src/components/organisms/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,75 @@
+import Arrow from "../../atoms/Buttons/Arrow";
+import React, { useState } from "react";
+import "./Carousel.scss";
+import { CSSTransition ,  TransitionGroup } from "react-transition-group";
+import ImageContainer from "./ImageContainer";
```

Comment:
This one is not used. Don't keep unused imports or variables in your codebase.

Reply:
Unnecessary code part deleted